packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -313,6 +313,16 @@ export const prodParasKusama: EndpointOption[] = [
       Parallel: 'wss://heiko-rpc.parallel.fi'
     }
   },
+  {
+    info: 'heiko',
+    homepage: 'https://parallel.fi',
+    paraId: 2126,
+    text: 'Heiko Crowdloan 2',
+    providers: {
+      OnFinality: 'wss://parallel-heiko.api.onfinality.io/public-ws',
+      Parallel: 'wss://heiko-rpc.parallel.fi'
```

Comment:
The same RPCs cannot point to 2 different para's - the runtime can only runt with 1 para at a time. If there are no providers, just leave them blank.